mix/ClientModel.h

```diff
@@ -243,12 +264,17 @@ private slots:
 	std::unique_ptr<RpcConnector> m_rpcConnector;
 	std::unique_ptr<Web3Server> m_web3Server;
 	std::shared_ptr<eth::FixedAccountHolder> m_ethAccounts;
+	std::unordered_map<Address, eth::Account> m_accounts;
+	std::vector<KeyPair> m_accountsSecret;
 	QList<u256> m_gasCosts;
 	std::map<std::pair<QString, int>, Address> m_contractAddresses;
 	std::map<Address, QString> m_contractNames;
 	std::map<QString, Address> m_stdContractAddresses;
 	std::map<Address, QString> m_stdContractNames;
 	CodeModel* m_codeModel = nullptr;
+	QList<QVariantList> m_queueTransactions;
+	QVariantMap m_currentScenario;
```

Comment:
This does not seem to be used except in one function. Also I'm not sure storing javascript objects is a good idea